content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -173,6 +173,12 @@ Vagrant typically assigns two interfaces to all VMs. The first, for which all ho
 
 This may lead to problems with flannel. By default, flannel selects the first interface on a host. This leads to all hosts thinking they have the same public IP address. To prevent this issue, pass the `--iface eth1` flag to flannel so that the second interface is chosen.
 
+It could be fixed with the add of an extra row in this file: /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+and add this extra arguments to kubernetes
+Environment="KUBELET_EXTRA_ARGS=--node-ip=<worker IP address>"
+
+if you are running other network you could have problem to the join of the node 
+
```

Comment:
the above copy and commit title have some typo and grammar problems.
most importantly, i don't think that `--node-ip` is a generic fix that would work for everyone.

Reply:
i see other people have solved with solution like this, so i think it was a good solution because it is static and everyone could see this in the config file

Reply:
could you provide reference links, please?